src/redux/App/reducer.js

```diff
@@ -13,6 +13,7 @@ const initState = new Map({
   appLocation: AppLocation.qtumPrediction,
   walletAddresses: [],
   lastUsedAddress: '',
+  changePassphraseResult: undefined,
```

Comment:
should be unnecessary if using `undefined` as default.

Reply:
yes, but it shows what's in the state. Also, probably better to use an empty string since that's what the type is right?

Reply:
Just like in components we would do
```
state = {
  changePassphraseResult: '',
}
```
if we don't do that in components it will error. Redux doesn't require it, but it's nice to follow the same pattern so we can see what's in the state.

Reply:
if thats the case, i suggest we use the pattern of `undefined` meaning not used, and `null` as not assigned and default value.

Reply:
the `changePassphraseResult` is an object  when we change passphrase.